### PR TITLE
Fix #176508 tutubox.io and app-valley.vip

### LINKED ISSUE #176508: tutubox.io
### Issue URL (Ads)
 [https://tutubox.io/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftutubox.io%2F)
Username: `@gototheskinny`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for iOS v4.5.2
System version: | ios 17.3.1
License type: | free
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Turkish

### PR DESCRIPTION
**Pull request by SpotlightForBugs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

- [x] Missed ads or ad leftovers;
- [x] Website or app doesn't work properly;
- [x] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [x] Filters maintenance.

### Enter the issue address

#176508

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

1. Your comment

https://github.com/AdguardTeam/AdguardFilters/issues/176508#issuecomment-2041549722

2. Screenshots

<details>

<summary>Screenshot 1 (tutubox issue):</summary>

image

</details>

<details>

<summary>Screenshot 2 (tutubox fix):</summary>

image

</details>

<details>

<summary>Screenshot 3/4 (app-valley.vip issue):</summary>

image
image

3: First ad popup not blocked at all
4: Second popup (leftovers)

</details>

<details>

<summary>Screenshot 5 (app-valley.vip fix) :</summary>

image

</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -4581,6 +4581,8 @@ wiki.gg##div[class^="sidebar-showcase"]
 wiki.gg##div[class^="games-showcase"]
 pornone.com##body .vjs-paused > .warp
 earth.com##div[class^="PrebidAdSlot_"]
+tutubox.io###vipModal > div.modal-content > iframe.vip-upsell__iframe:last-child
+store.app-valley.vip#$?#.modal1 { remove: true; }
```
Comment (Alex-302) [inline BaseFilter/sections/specific.txt:4585]: On my end `tutubox.io,store.app-valley.vip###vipModal` works fine.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (SpotlightForBugs) [reply BaseFilter/sections/specific.txt:4585]: It does? That's odd

Comment (SpotlightForBugs) [reply BaseFilter/sections/specific.txt]: You're right!!!, changed it to your not css-based Solution.

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
Thanks

**Pull request merged**